src/main/java/com/gluonhq/substrate/model/ProjectConfiguration.java

```diff
@@ -125,10 +126,19 @@ public Path getJavaStaticLibsPath() {
      * @return the path to the JavaFX SDK (including at least the libs)
      */
     public Path getJavafxStaticPath() {
+        String depsTarget;
+        switch (targetTriplet.getOs()) {
+            case Constants.OS_LINUX: depsTarget = Constants.DEPS_TARGET_LINUX; break;
+            case Constants.OS_DARWIN: depsTarget = Constants.DEPS_TARGET_MAC; break;
+            case Constants.OS_IOS: depsTarget = Constants.DEPS_TARGET_IOS; break;
+            default: throw new RuntimeException("OS for target " + targetTriplet + " not supported");
+        }
+
         Path answer = Constants.USER_SUBSTRATE_PATH
                 .resolve("javafxStaticSdk")
                 .resolve(getJavafxStaticSdkVersion())
-                .resolve(targetTriplet.getOsArch());
+                .resolve(targetTriplet.getOsArch())
+                .resolve(depsTarget + "-sdk");
```

Comment:
the os-arch combination already uniquely identifies the possible version, no need to add "depsTarget" here